pkg/scorecard.go

```diff
@@ -160,6 +162,23 @@ func runScorecard(ctx context.Context,
 	// If the user runs checks
 	go runEnabledChecks(ctx, repo, request, checksToRun, resultsCh)
 
+	// Get configuration
+	rc, err := repoClient.GetFileReader("scorecard.yml")
+	// If configuration file exists, continue. Otherwise, ignore
+	if err == nil {
+		defer rc.Close()
+		checks := []string{}
+		for check := range checksToRun {
+			checks = append(checks, check)
+		}
+		c, err := config.Parse(rc, checks)
+		if err != nil {
+			logger := sclog.NewLogger(sclog.DefaultLevel)
+			logger.Error(err, "parsing configuration file")
+		}
+		ret.Config = c
+	}
+
```

Comment:
Please wrap this all in an experimental check, otherwise LGTM

Reply:
for probes we had something like:
```go
if os.Getenv(options.EnvVarScorecardExperimental) == "1"
```

Reply:
Ack

Reply:
Done